.gitignore

```diff
@@ -132,3 +114,7 @@ com_crashlytics_export_strings.xml
 crashlytics.properties
 crashlytics-build.properties
 fabric.properties
+
+# Ignore stuff generated by the example
```

Comment:
Next 2 lines are already ignored:
1. Under the [develop](https://github.com/arduino-cmake/arduino-cmake/blob/develop/.gitignore) branch - There's a full ignore of several CMake files
2. In line number 90 in this same file, and also in the **develop** branch as mentioned above.